app/data_migrations/delete_invoice_with_fein.rb

```diff
@@ -37,7 +37,7 @@ def delete_initial_employer_invoice_notice(orgs)
 
   def delete_message_from_inbox(orgs)
     orgs.each do |org|
-      messages = org.employer_profile.inbox.messages.select{|message| message.subject == "Your Invoice is Now Available in your DC Health Link Account" }
+      messages = org.employer_profile.inbox.messages.select{|message| message.subject == "Your Invoice is Now Available in your Cover ME Account" }
```

Comment:
An `.rb` file, so we should reference Settings name in the `disable_shop_base` branch.